library/std/src/env.rs

```diff
@@ -65,21 +65,17 @@ pub fn set_current_dir<P: AsRef<Path>>(path: P) -> io::Result<()> {
 
 /// An iterator over a snapshot of the environment variables of this process.
 ///
-/// This structure is created by the [`std::env::vars`] function. See its
-/// documentation for more.
-///
-/// [`std::env::vars`]: vars
+/// This structure is created by the [`vars`] function. See its documentation
+/// for more.
```

Comment:
_We need to go deeper._
```suggestion
/// This structure is created by [`vars()`]. See its documentation for more.
```

Reply:
I like this.